pulp_smash/tests/rpm/cli/test_environments.py

```diff
@@ -0,0 +1,70 @@
+# coding=utf-8
+"""Tests for RPM package environments."""
+from __future__ import unicode_literals
+
+import subprocess
+import unittest2
+
+from pulp_smash import cli, config, utils
+from pulp_smash.constants import (
+    RPM_ENV_DESC,
+    RPM_ENV_NAME,
+    RPM_FEED_URL,
+)
+
+
+class UploadPackageEnvTestCase(unittest2.TestCase):
+    """Test whether Pulp can upload package environments into a repository.
+
+    This test case covers `Pulp # 1003`_ and the corresponding Pulp Smash
```

Comment:
Would be great to also link to the documentation http://docs.pulpproject.org/plugins/pulp_rpm/user-guide/recipes.html#create-your-own-package-environment